utils/data_cleaning.py

```
@@ -0,0 +1,63 @@
+"""
+🧹 Data Cleaning Utilities
+Consolidated from logic.py, table_one.py, and poisson_lib.py
+Driven by central configuration from config.py
+"""
+
+import pandas as pd
+import numpy as np
+from typing import Any, Union
+from config import CONFIG  # นำเข้าตัวสั่งการ
+
+def clean_numeric(val: Any) -> float:
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `val`

(ANN401)